src/src_main/sadb_routine_inmemory.template.c

```diff
@@ -386,6 +413,7 @@ static int32_t sadb_get_operational_sa_from_gvcid(uint8_t tfvn, uint16_t scid, u
 
         for (i = 0; i < NUM_SA; i++)
         {
+            printf("LOOP: %d\n", i);
```

Comment:
A debug state without an #ifdef DEBUG, delete this -- try not to commit these.

Reply:
Good catch.  Seems to be a habit of mine to miss my debugging prints during cleanup.